Error compiling on aarch64

Trying to compile nimbus ```v23.7.0``` on a Linux Fedora 37 guest virtual machine on top of MacBook Pro M2 host, using parallels virtualization. So it's essentially on a AARCH64 Linux with native optimisations disabled.

```
make NIMFLAGS="-d:disableMarchNative" -j8 nimbus_beacon_node
```

Getting error:

```
/home/user/nimbus/vendor/nim-confutils/confutils.nim(935, 23) Warning: Cannot prove that 'result' is initialized. This will become a compile time error in the future. [ProveInit]

/tmp/ccRV4lza.ltrans50.ltrans.o: in function `getMetadataForNetwork__networkingZnetwork95metadata_897':
/home/user/nimbus/beacon_chain/networking/network_metadata.nim:296:(.text+0x1087c): relocation truncated to fit: R_AARCH64_LDST64_ABS_LO12_NC against symbol `eth2_sepolia_genesis_size' defined in .rodata section in /home/user/nimbus/nimcache/release/nimbus_beacon_node/@mnetworking@snetwork_metadata_mainnet.S.o
/usr/bin/ld: /home/user/nimbus/beacon_chain/networking/network_metadata.nim:296: warning: one possible cause of this error is that the symbol is being referenced in the indicated code as if it had a larger alignment than was declared where it was defined
/home/user/nimbus/beacon_chain/networking/network_metadata.nim:296:(.text+0x10a3c): relocation truncated to fit: R_AARCH64_LDST64_ABS_LO12_NC against symbol `eth2_goerli_genesis_size' defined in .rodata section in /home/user/nimbus/nimcache/release/nimbus_beacon_node/@mnetworking@snetwork_metadata_mainnet.S.o
/usr/bin/ld: /home/user/nimbus/beacon_chain/networking/network_metadata.nim:296: warning: one possible cause of this error is that the symbol is being referenced in the indicated code as if it had a larger alignment than was declared where it was defined
collect2: error: ld returned 1 exit status
make[1]: *** [nimcache/release/nimbus_beacon_node/nimbus_beacon_node.makefile:3152: build] Error 1
make: *** [Makefile:437: nimbus_beacon_node] Error 2
```

The same command was working fine for all previous Nimbus versions

--- a/beacon_chain/networking/network_metadata_gnosis.S
+++ b/beacon_chain/networking/network_metadata_gnosis.S
@@ -31,6 +31,7 @@ gnosis_mainnet_genesis_data:
     .incbin "gnosis-chain-configs/mainnet/genesis.ssz"
 gnosis_mainnet_genesis_end:
     .global cdecl(gnosis_mainnet_genesis_size)
+    .p2align 3
 cdecl(gnosis_mainnet_genesis_size):
     .quad    gnosis_mainnet_genesis_end - gnosis_mainnet_genesis_data
 
@@ -38,6 +39,7 @@ gnosis_chiado_genesis_data:
     .incbin "gnosis-chain-configs/chiado/genesis.ssz"
 gnosis_chiado_genesis_end:
     .global cdecl(gnosis_chiado_genesis_size)
+    .p2align 3
 cdecl(gnosis_chiado_genesis_size):
     .quad    gnosis_chiado_genesis_end - gnosis_chiado_genesis_data
 

--- a/beacon_chain/networking/network_metadata_mainnet.S
+++ b/beacon_chain/networking/network_metadata_mainnet.S
@@ -31,6 +31,7 @@ eth2_mainnet_genesis_data:
     .incbin "eth2-networks/shared/mainnet/genesis.ssz"
 eth2_mainnet_genesis_end:
     .global cdecl(eth2_mainnet_genesis_size)
+    .p2align 3
 cdecl(eth2_mainnet_genesis_size):
     .quad    eth2_mainnet_genesis_end - eth2_mainnet_genesis_data
 
@@ -38,6 +39,7 @@ eth2_goerli_genesis_data:
     .incbin "eth2-networks/shared/prater/genesis.ssz"
 eth2_goerli_genesis_end:
     .global cdecl(eth2_goerli_genesis_size)
+    .p2align 3
 cdecl(eth2_goerli_genesis_size):
     .quad    eth2_goerli_genesis_end - eth2_goerli_genesis_data
 
@@ -45,6 +47,7 @@ eth2_sepolia_genesis_data:
     .incbin "sepolia/bepolia/genesis.ssz"
 eth2_sepolia_genesis_end:
     .global cdecl(eth2_sepolia_genesis_size)
+    .p2align 3
 cdecl(eth2_sepolia_genesis_size):
     .quad    eth2_sepolia_genesis_end - eth2_sepolia_genesis_data
 

